src/data/privacy_de.json

```diff
@@ -8,13 +8,32 @@
+                "Sollte ein Gast später Corona-positiv getestet werden und über die App eine Warnung auslösen, werden alle Gäste gewarnt, die zeitgleich bei Ihrem Event oder Ort eingecheckt waren. Die App Ihres Gastes übermittelt in diesem Fall nur die Event-ID an das Serversystem.",
+                "Wenn einer Ihrer Gäste Corona-positiv getestet wurde und andere Gäste nicht mittels der Corona-Warn-App gewarnt hat (zum Beispiel, weil er nicht eingecheckt war oder weil er den Eintrag zu dem Event oder Ort nachträglich gelöscht hat), kann das zuständige Gesundheitsamt Ihnen als Gastgeber eine TAN mitteilen, damit Sie die eingecheckten Gäste mittels der Corona-Warn-App warnen.",
+                "Alle im Rahmen der QR-Code-Erstellung angegebenen Informationen werden ausschließlich lokal in Ihrem Browser verarbeitet. Weder die von Ihnen generierten QR-Codes noch die darin enthaltenen Informationen werden an die Server der Corona-Warn-App gesendet. Eine Datenverarbeitung durch das RKI findet bei der Erstellung eines Event QR-Codes nicht statt.",
+                "<b>4. Ihre Datenschutzrechte (Rechte der betroffenen Person)</b>",
+                "Soweit das RKI personenbezogene Daten von Ihnen verarbeitet, stehen Ihnen unter den gesetzlichen Voraussetzungen folgende Datenschutzrechte zu:",
+                "<ul><li>das Recht, jederzeit Auskunft über die Verarbeitung Ihrer personenbezogenen Daten zu erhalten (Art. 15 DSGVO),</li><li>das Recht auf Berichtigung unrichtiger Daten oder Vervollständigung lückenhafter Daten (Art. 16 DSGVO),</li><li>das Recht, Daten nach den gesetzlichen Vorgaben löschen oder in der Verarbeitung einschränken zu lassen (z.&nbsp;B. bei Widerruf Ihrer Einwilligung oder unrechtmäßiger Verarbeitung) (Art. 17, 18 DSGVO),</li><li>das Recht, bei einer auf Einwilligung beruhenden Datenverarbeitung Ihre Einwilligung jederzeit mit Wirkung für die Zukunft zu widerrufen (Art. 7 Abs. 3 DSGVO),</li><li>das Recht auf Datenübertragbarkeit (Sie können eine Übersicht Ihrer Daten in einem elektronischen Format zur Verfügung gestellt bekommen) (Art. 20 DSGVO),</li><li><strong>das Recht auf Widerspruch gegen die Datenverarbeitung, die aus Gründen eines berechtigten Interesses des RKI, für die Wahrnehmung von öffentlichen Aufgaben oder in Ausübung öffentlicher Gewalt erfolgt (Art. 21 DSGVO)</strong></li><li>das Recht, den Datenschutzbeauftragten des RKI zu kontaktieren und Ihr Anliegen vorzubringen (Art. 38 Abs. 4 DSGVO) und</li><li>das Recht, sich bei einer Aufsichtsbehörde für den Datenschutz zu beschweren (z.&nbsp;B. der Bundesbeauftragte für den Datenschutz und die Informationsfreiheit, Graurheindorfer Str. 153, 53117 Bonn, <a href='tel: +492289977990'>+49 228-997799-0</a>, E-Mail: <a href='poststelle...bfdi...bund...de' class='email'>poststelle...bfdi...bund...de</a>, <a href='http://www.bfdi.bund.de' target='_blank' rel='noopener noreferrer'>http://www.bfdi.bund.de</a>) (Art. 77 Abs. 1 DSGVO).</li></ul>",
```

Comment:
Comma missing at end of line
"das Recht auf Widerspruch gegen die Datenverarbeitung, die aus Gründen eines berechtigten Interesses des RKI, für die Wahrnehmung von öffentlichen Aufgaben oder in Ausübung öffentlicher Gewalt erfolgt (Art. 21 DSGVO)"